lib/service/lru.js

```diff
@@ -0,0 +1,55 @@
+class LRU {
+  constructor(crowi) {
+    this.crowi = crowi
+    this.max = 10
+    this.client = null
+
+    if (this.crowi.redisOpts) {
+      const redis = require('redis')
+      const redisClient = redis.createClient(this.crowi.redisOpts)
+      this.client = redisClient
```

Comment:
❓ Is there any reason why you didn't use `crowi.redis` client?

Reply:
No reason in particular. I change this service to reuse a connection.